lib/error-message.js

```diff
@@ -22,13 +22,13 @@ class CustomError extends Error {
 }
 
 /**
- * @param {Options} options
+ * @param {Options | null} options
  * @param {CustomError} err
  * @param {Path} [defaultPath]
  * @returns {string}
  */
 function report(options, err, defaultPath) {
-  if (options.report === 'json') {
+  if (options && options.report === 'json') {
```

Comment:
I'm surprised, are there ever instances where options is null? 🤔

Reply:
There was a comment to fix it later:

https://github.com/jfmengels/node-elm-review/blob/9fa20ed5cec3e237fa205bcfe7c714fe157d7600/lib/options.js#L543-L545

It wasn't null, but an empty object. Since the `reportErrorAndExit` function didn't get any `Options` as args, I thought `null` would be better than `{}`.

But maybe that wasn't what you meant to do! I can always revert back, make `report` accept only `Options` (as opposed to `Options | null`), and put the comment back

Reply:
AH yes. It seems for that location I was pretty lazy back when I added it. This works for me I think :+1: